src/Drawer/Drawer.js

```diff
@@ -72,6 +72,7 @@ export default class Drawer extends Component {
     docked: PropTypes.bool,
     enterTransitionDuration: PropTypes.number,
     leaveTransitionDuration: PropTypes.number,
+    onRequestClose: PropTypes.func,
```

Comment:
I am not sure about this one. It is true that additional props specified will be passed to Modal, but that means for most basic use case (controlled open/close Drawer) you have to know implementation internals of Drawer. Which is why i included this explicit prop, which is only passed.